packages/p2p/src/stores/general-store.js

```diff
@@ -130,6 +130,7 @@ export default class GeneralStore extends BaseStore {
             } else {
                 this.setAdvertiserId(p2p_advertiser_create.id);
                 this.setIsAdvertiser(!!p2p_advertiser_create.is_approved);
+                this.setIsListed(p2p_advertiser_create.is_listed);
```

Comment:
the subscribe-able `p2p_advertiser_info` does not update as soon as a new user is created, thus `is_listed_` is not updated yet. Response from creating a new user `p2p_advertiser_create` has `is_listed` field, thus manually updated this observable for this case when creating a new user after entering their nickname